### ISSUE TITLE
Timestamp related fields null/empty when postgres datestyle configuration is different from default

### ISSUE THREAD

**Issue by bsagarbhat** [Unleash/unleash#1670]
### Describe the bug

Timestamp related fields null/empty when postgres datestyle configuration is different from default.

Works with default configuration of postgres which is 'iso, mdy'. 

It doesn't work when server postgres timestamp is changed to 'Postgres, DMY' or 'redwood,show_time'

API Endpoints such as 

/api/admin/features returns created_at and last_seen_at as null 

/api/admin/projects/default/health-report -- Fails with http status code 500

### Steps to reproduce the bug

1. Start postgres in docker container with datestyle passed as 'Postgres, DMY'. . `docker  run -d --name unleash-postgres -p 5432:5432 -e POSTGRES_USER=unleash -e POSTGRES_PASSWORD=unleash  artifactory.idfcbank.com/docker/postgres:14.2 -c timezone='Asia/Kolkata' -c datestyle='Postgres, DMY'`
2. Start Unleash
3. Create Sample Toggle
4. Get Toggle details using GET endpoint. /api/admin/features 
5. Output as created_at and last_seen_at as null.
```json 
{
  "version": 1,
  "features": [
    {
      "strategies": [
        
      ],
      "impressionData": false,
      "enabled": false,
      "name": "TestToggle",
      "description": "",
      "project": "default",
      "stale": false,
      "type": "release",
      "variants": [
        
      ],
      "lastSeenAt": null,
      "createdAt": null
    }
  ]
}
```

### Expected behavior

I expected created at to be returned as follows.

```json
{
  "version": 1,
  "features": [
    {
      "strategies": [
        
      ],
      "impressionData": false,
      "enabled": false,
      "name": "TestToggle",
      "description": "",
      "project": "default",
      "stale": false,
      "type": "release",
      "variants": [
        
      ],
      "lastSeenAt": null,
      "createdAt": "2022-06-06T14:19:14.324Z"
    }
  ]
}
```

### Unleash version

4.8.2

### Subscription type

Open source

### Hosting type

Self-hosted

**Comment by sighphyre** [comment]
That's so much for the very thorough bug report, I'll raise this with the team today and get back to you soon!

**Comment by sighphyre** [comment]
Hi @bsagarbhat, we've spoken a little about this one internally. While this is definitely a missing feature, we don't have plans to add this in this quarter. Just to understand, from your perspective is this a nice thing to have or is this absolutely blocking you from using Unleash?

**Comment by palmski** [comment]
FWIW we too have come across this problem. I solved it on 3.11 by adding this code
```
unleash.start(localConfig).then(unleash => {
    // Override the datestyle in the connection session to match what the server is producing - NB this might not be the best way to do this
    unleash.stores.db.client.pool.on('acquireSuccess', (eventId, resource) => {
        resource.query('set datestyle to \'ISO, DMY\'', (err, res) => {
            if(err) {
                console.log(err)
            }
        })
    })
```
I'm trying to replicate the fix with 4.12 but getting a handle on the Knex object seems harder

**Comment by sighphyre** [comment]
@palmski We no longer expose the Knex object in the v4 range of Unleash so I don't think that'll work anymore. If you're willing to hold off on upgrading until 4.13 is released, we'll have something for you then

**Comment by sighphyre** [comment]
This has been implemented and released in 4.13, closed by #1689

There's some details here: https://docs.getunleash.io/deploy/configuring_unleash_v3 on the `ALLOW_NON_STANDARD_DB_DATES` env var that you can use to set this
